<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Kajetan Åšwerik</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>534</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Morph OpenCL</string>
  </property>
  <widget class="QWidget" name="qcentralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QStatusBar" name="qstatusBar"/>
  <widget class="QMenuBar" name="qmenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuFromCamera">
     <property name="title">
      <string>From camera</string>
     </property>
     <addaction name="actionCameraUseOpenCV"/>
     <addaction name="actionCameraUseSapera"/>
    </widget>
    <addaction name="menuFromCamera"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSaveSE"/>
    <addaction name="actionOpenSE"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Configure</string>
    </property>
    <addaction name="actionOpenCL"/>
    <addaction name="actionPickMethod"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionUse_OpenCL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use OpenCL</string>
   </property>
  </action>
  <action name="actionOpenCL_Settings">
   <property name="text">
    <string>OpenCL Settings</string>
   </property>
  </action>
  <action name="actionOpenCL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Use OpenCL</string>
   </property>
   <property name="toolTip">
    <string>Use OpenCL instead of OpenCV</string>
   </property>
  </action>
  <action name="actionPickMethod">
   <property name="text">
    <string>Pick method</string>
   </property>
   <property name="toolTip">
    <string>Pick method again (buffer 1D or 2D)</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveSE">
   <property name="text">
    <string>Save structuring element</string>
   </property>
  </action>
  <action name="actionOpenSE">
   <property name="text">
    <string>Open structuring element</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCameraUseSapera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Use Sapera</string>
   </property>
  </action>
  <action name="actionCameraUseOpenCV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use OpenCV</string>
   </property>
  </action>  
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
